src/components/modals/data-table-modal.js

```diff
@@ -30,183 +32,179 @@ import {Clock} from 'components/common/icons/index';
 // Breakpoints
 import {media} from 'styles/media-breakpoints';
 
-const ReactDataGrid = window.navigator ? require('react-data-grid/dist/react-data-grid.min') : null;
-
-let shouldPreventScrollBack = false;
-
-if (window.navigator && window.navigator.userAgent) {
-  const {navigator} = window;
-  // Detect browsers
-  // http://stackoverflow.com/questions/5899783/detect-safari-using-jquery
-  const isMac = navigator.userAgent.match(/Macintosh/);
-  const is_chrome = navigator.userAgent.indexOf('Chrome') > -1;
-  const is_safari = navigator.userAgent.indexOf('Safari') > -1;
-  const is_firefox = navigator.userAgent.indexOf('Firefox') > -1;
+const COLUMN_SIZE = {
+  [ALL_FIELD_TYPES.timestamp]: 200,
+  [ALL_FIELD_TYPES.date]: 150,
+  [ALL_FIELD_TYPES.point]: 150,
+  [ALL_FIELD_TYPES.string]: 150
+};
 
-  // prevent chrome scroll back
-  shouldPreventScrollBack = isMac && (is_chrome || is_safari || is_firefox);
-}
+const DEFAULT_COLUMN_WIDTH = 100;
+const HEADER_HEIGHT = 72;
 
 const dgSettings = {
-  sidePadding: '38px'
+  sidePadding: 36,
+  verticalPadding: 16,
+  height: 36
 };
 
-const DataGridWrapper = styled.div`
-  .react-grid-Main {
-    outline: 0;
-  }
-
-  .react-grid-Grid {
-    border: 0;
-  }
-
-  .react-grid-Cell {
-    border-right: 0;
-    border-bottom: ${props => props.theme.panelBorderLT};
-    padding-left: 16px;
```

Comment:
Left most of cell and header cell should have equal padding. Now it's too close to the edge
<img width="230" alt="Screen Shot 2019-06-21 at 12 41 27 AM" src="https://user-images.githubusercontent.com/3605556/59906366-6db21480-93bd-11e9-899b-f418a5534ea5.png">